sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -139,18 +142,6 @@ We are cleaning the public API surface by
 - Updated to use the latest version of the `rhea` package.
   This update improves support for [bundling](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Bundling.md) this library.
 
-## 1.0.0 (2019-01-08)
```

Comment:
I don't like this change...though I think it's because the date is wrong. 1.0.0 was probably released on 2020-01-08, not 2019-01-08

Reply:
@chradek Can you push a commit to this PR fixing this? The automation must have been doing the right thing, but our previous changelog entry had the wrong date like you mentioned

Reply:
Updated!